package.json

```diff
@@ -27,8 +27,15 @@
     "amqp-coffee": "0.1.30",
     "async": "2.1.5",
     "debug": "2.6.3",
-    "lodash": "4.17.4",
+    "lodash.clone": "^4.5.0",
     "lodash.get": "4.4.2",
+    "lodash.isfunction": "^3.0.8",
```

Comment:
here is a link https://github.com/lodash/lodash/wiki/Build-Differences

If you think this is still to big, I'm ok with this.